x/feemarket/migrations/v010/migrate_test.go

```diff
@@ -22,12 +21,12 @@ import (
 func TestMigrateStore(t *testing.T) {
 	encCfg := encoding.MakeConfig(app.ModuleBasics)
 	feemarketKey := sdk.NewKVStoreKey(feemarkettypes.StoreKey)
-	tFeeMarketKey := sdk.NewTransientStoreKey(fmt.Sprintf("%s_test", feemarkettypes.StoreKey))
+	tFeeMarketKey := sdk.NewTransientStoreKey("margetkey_test")
```

Comment:
?

Reply:
`feemarkettypes.StoreKey` is already stored, It is storing the duplicate keys, thats why I added new StoreKey